### PR TITLE
BUILD-5553 Changing Vault key names du to refactoring

### PR DESCRIPTION
**Pull request by SamirM-BE**
Please review our [contribution guidelines](https://github.com/SonarSource/sonarlint-intellij/blob/master/contributing.md).

And please ensure your pull request adheres to the following guidelines: 

- [ ] Please explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [ ] Use the following formatting style: [SonarSource/sonar-developer-toolset](https://github.com/SonarSource/sonar-developer-toolset#code-style)
- [ ] Provide a unit test for any code you changed
- [ ] If there is a [JIRA](http://jira.sonarsource.com/browse/SLI) ticket available, please make your commits and pull request start with the ticket ID (SLI-XXXX)

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -214,8 +214,8 @@ qa_task:
     cpu: 6
     memory: 16G
   env:
-    CLION_KEY: VAULT[development/team/sonarlint/kv/data/clion.key data.key]
-    GOLAND_KEY: VAULT[development/team/sonarlint/kv/data/goland.key data.key]
+    CLION_KEY: VAULT[development/team/sonarlint/kv/data/ide.keys clion]
```
Comment (tomverin) [inline .cirrus.yml:217]: it's data.clion here

File: .cirrus.yml [hunk 2]
```diff
@@ -214,8 +214,8 @@ qa_task:
...
+    GOLAND_KEY: VAULT[development/team/sonarlint/kv/data/ide.keys goland]
```
Comment (tomverin) [inline .cirrus.yml:218]: it's data.goland
The KV secrets engine returns an array called data to cirrus, thus you need to get the key goland in the array data

**Review by tomverin — CHANGES_REQUESTED** [review]

File: .cirrus.yml (hunk 1 above)
Comment (SamirM-BE) [reply .cirrus.yml]: done

File: .cirrus.yml (hunk 2 above)
Comment (SamirM-BE) [reply .cirrus.yml]: done

**Review by tomverin — APPROVED** [review]

**Review by nquinquenel — APPROVED** [review]
Thank you!

**Pull request merged**